Tests/XCUITests/BookmarksTests.swift

```diff
@@ -206,10 +206,20 @@ class BookmarksTests: BaseTestCase {
     }
 
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2303043
+    // Test failing in M1s because the swipe gesture. Needs work to run only on Intel.
     func testDeleteBookmarkSwiping() {
         addNewBookmark()
         // Remove by swiping
-        app.tables["Bookmarks List"].staticTexts["BBC"].swipeLeft()
+        if processIsTranslatedStr() != m1Rosetta {
+            app.tables["Bookmarks List"].staticTexts["BBC"].swipeLeft()
+            mozWaitForElementToExist(app.buttons["Delete"])
+            app.buttons["Delete"].tap()
+        }
```

Comment:
If I'm not mistaken...this should be added in an else {} statement, otherwise for Intel, the test will remove a bookmark and then it will fail in trying to delete it again

Reply:
you are right, added the else statement.